packages/client/hmi-client/src/components/workflow/scenario-templates/scenario-template.ts

```diff
@@ -0,0 +1,23 @@
+import { Workflow } from '@/types/workflow';
+import { useProjects } from '@/composables/project';
+import { AssetType } from '@/types/Types';
+
+export abstract class BaseScenarioTemplate {
+	templateId: string;
+
+	templateName: string;
+
+	description: string;
+
+	workflowName: string;
+
+	constructor(nameId: string, name: string, description: string) {
+		this.templateId = nameId;
+		this.templateName = name;
+		this.description = description;
+		const workflows = useProjects().getActiveProjectAssets(AssetType.Workflow);
+		this.workflowName = `workflow ${workflows.length + 1}`;
+	}
```

Comment:
This feels a bit awkward
- The workflowName looks like it would be provided from the design sketch, so I don't think we need to generate a name here
- It doesn't really make sense to have both templateId and templateName, it should probably be one or the other, but not both

Reply:
I've updated `templateId` and `templateName` to be static variables on each class as they are used for the radio button list on the left side

Reply:
For workflow name, I can change that...I figure we can just have the same behaviour we have now where the default name is the workflows.length + 1

Reply:
I agree with @mwdchang, the name should be the one provided by the template.
If I use the template named `Extract model from Document`, I expect my new template to be named the same.

Reply:
Which make the name not part of the `BaseScenario` class

Reply:
Sure, I've changed the name to be unique to each template